ci-operator/step-registry/cucushift/installer/check/ibmcloud/destroy/cucushift-installer-check-ibmcloud-destroy-ref.yaml

```diff
@@ -0,0 +1,24 @@
+ref:
+  as: cucushift-installer-check-ibmcloud-destroy
+  from_image:
+    namespace: ocp
+    name: "4.13"
+    tag: upi-installer
+  grace_period: 10m
+  commands: cucushift-installer-check-ibmcloud-destroy-commands.sh
+  resources:
+    requests:
+      cpu: 10m
+      memory: 100Mi
+  env:      
+  - name: IBMCLOUD_DNS_INSTANCE_NAME
+    default: ""
+    documentation: |-
+      IBMCloud DNS instance name where BASE_DOMAIN resides, it is required when installing a private cluster,
+      adding private DNS record for mirror registry.
```

Comment:
`adding private DNS record for mirror registry`? I do not think this is the core purpose for the mirror-registry DNS, the core purpose of the dns instance is for cluster creation, api and apps urls, mirror-registry dns is just re-use the dns.